### ISSUE TITLE
Gem::Security::Exception

### ISSUE THREAD

**Issue by christoff-1992** [bkuhlmann/git-cop#35]
Following the instructions to first install the certificate and then install via:

1.`gem cert --add <(curl --location --silent https://www.alchemists.io/gem-public.pem)`
2.`gem install git-cop --trust-policy MediumSecurity`

Still causes the following error to occur:

```
ERROR:  While executing gem ... (Gem::Security::Exception)
    no digests provided (probable bug)
```

Can confirm the certificate hasn't expired, have tried to manually import the certificate too without success. (High Sierra if it makes a difference)

**Comment by koenrh** [comment]
I think this particular issue is caused by an outdated 'rubygems' package. Try updating 'rubygems': `gem update --system`.

But there seems to be another issue that prevents you from installing the gem. See: https://github.com/bkuhlmann/git-cop/issues/36.

**Comment by christoff-1992** [comment]
Ahh brilliant will take a look at both and get back to you.

Thanks

**Comment by bkuhlmann** [comment]
Thanks for reporting this. This has been fixed in Issue #36. These changes are available in [Version 1.7.1](https://github.com/bkuhlmann/git-cop/releases/tag/v1.7.1) which is available for download now.
